### DISCUSSION TITLE
NPM browser shot error with laravel pdf package

### DISCUSSION THREAD

**Discussion by ShaungBhone** [spatie/laravel-pdf#185] [Q&A]
I got an error with spatie laravel pdf. I already installed all requirement from docs. I'm using
```
- Mac
- Safari
- Herd
```

```The command "PATH=$PATH:/usr/local/bin:/opt/homebrew/bin NODE_PATH=`npm root -g` node '/Users/shaungbhone/Herd/weshop/vendor/spatie/browsershot/src/../bin/browser.cjs' '{"url":"file:\/\/\/var\/tmp\/89399023-0225763001727011898\/index.html","action":"pdf","options":{"path":"inovices.pdf","args":[],"viewport":{"width":800,"height":600},"displayHeaderFooter":false,"printBackground":true}}'" failed. Exit Code: 127(Command not found) Working directory: /Users/shaungbhone/Herd/weshop/public Output: ================ Error Output: ================ sh: npm: command not found sh: node: command not found```

**Comment by sakaye** [comment]
Have you installed Puppeteer?
`npm i -g puppeteer`

https://spatie.be/docs/browsershot/v4/requirements

**Reply by ShaungBhone** [reply]
Yeah. All are working fine in my local development.

**Reply by ShaungBhone** [reply]
I mean that https://github.com/spatie/laravel-pdf/issues/187
